Update graphql service BBEs description (#3757)
ballerina-platform/ballerina-distribution: examples/graphql-interfaces/graphql_interfaces.md

# GraphQL service - Interfaces

> A GraphQL schema can have interfaces. An interface specifies a set of fields that multiple object types can include. In Ballerina, interfaces are defined using `distinct` `service` objects, and the fields of the interfaces are defined as resource method definitions. Objects that are implementing the interfaces must implement the `resource` methods defined in the service objects. The Ballerina type inclusion includes the interface type to an object type. Use an interface when you want to return a type that consists of a set of possible types with common fields.

Reviewer: ```suggestion
A GraphQL schema can have interfaces. An interface specifies a set of fields that multiple object types can include. In Ballerina, interfaces are defined using `distinct` `service` objects, and the fields of the interfaces are defined as resource method definitions. Objects that are implementing the interfaces must implement the `resource` methods defined in the service objects. The Ballerina type inclusion includes the interface type to an object type. Use an interface to return a type that consists of a set of possible types with common fields.
```

> A GraphQL schema can have interfaces. An interface specifies a set of fields that multiple object types can include. In Ballerina, interfaces are defined using `distinct` `service` objects and the fields of the interfaces are defined as resource method definitions. Objects that are implementing the interfaces must implement the `resource` methods defined in the service objects. The Ballerina type inclusion is used to include the interface type to an object type. Use an interface when you want to return an object or set of objects, but those might be of several different types.

Reviewer 2: ```suggestion
A GraphQL schema can have interfaces. An interface specifies a set of fields that multiple object types can include. In Ballerina, interfaces are defined using `distinct` and `service` objects, and the fields of the interfaces are defined as resource method definitions. Objects that are implementing the interfaces must implement the `resource` methods defined in the service objects. The Ballerina type inclusion includes the interface type to an object type. Use an interface to return an object or set of objects. However, those might be of several different types.
```

Author: Do we need to have **and** between `distinct` `service`?  
I think we have to read it as "`distinct` `service` objects" similar to what we write in ballerina code
ex: `type Profile distinct service object { }`

Author: Updated other suggestions

Reviewer: I don't think we need that. +1 to use `distinct` `service` objects.
